samples/app-checkin-location/nodejs/index.js

```diff
@@ -46,6 +47,8 @@ adapter.onTurnError = async (context, error) => {
     // Send a message to the user
     await context.sendActivity('The bot encountered an error or bug.');
     await context.sendActivity('To continue to run this bot, please fix the bot source code.');
+     // Uncomment this for local debugging.
+    //  await context.sendActivity(`Sorry, it looks like something went wrong. Exception Caught: ${error}`);
```

Comment:
// await context.sendActivity(`Sorry, it looks like something went wrong. Exception Caught: ${error}`);